docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -255,7 +255,7 @@ datasources:
 +
 [source,shell]
 ----
-./elastic-agent -c elastic-agent-standalone.yml run
+./elastic-agent -c elastic-agent.yml run
```

Comment:
In this case I believe `./elastic-agent run` is sufficient, as it uses `elastic-agent.yml` by default?

I'm fine with the change. That said, the reason to use a different filename here was to prevent the configuration used for standalone mode to be overwritten when the user later switches to fleet mode.

I know there's a warning that the config will be overwritten, so either way is fine with me.

Reply:
Yes, we also want to reduce the friction and concepts.